### DISCUSSION TITLE
Coding problems.

### DISCUSSION THREAD

**Discussion by udscminidisc** [NovaoTech/solaris#17] [Q&A] [answered]
I don't know how to code, where can I start?

**Comment by MikeDev101** [comment, marked as answer]
We're primarily using Python for the backend, HTML/CSS/JS (?) for the frontend. Try finding an introductory online tutorial for those languages respectively.

**Reply by udscminidisc** [reply]
Thanks.

**Reply by icarusinthewings** [reply]
@yoo832, We are using the Flask framework.

**Reply by udscminidisc** [reply]
Ok.

**Reply by udscminidisc** [reply]
I got html down!
